twitter.html

```diff
@@ -0,0 +1,458 @@
+                </li>
+                <li>
+                  <p>
+                    <svg
+                      xmlns="http://www.w3.org/2000/svg"
+                      class="h-6 w-6"
+                      fill="none"
+                      viewBox="0 0 24 24"
+                      stroke="currentColor"
+                    >
+                      <path
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
+                        stroke-width="2"
+                        d="M15 17h5l-1.405-1.405A2.032 2.032 0 0118 14.158V11a6.002 6.002 0 00-4-5.659V5a2 2 0 10-4 0v.341C7.67 6.165 6 8.388 6 11v3.159c0 .538-.214 1.055-.595 1.436L4 17h5m6 0v1a3 3 0 11-6 0v-1m6 0H9"
+                      />
+                    </svg>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    <svg
+                      xmlns="http://www.w3.org/2000/svg"
+                      class="h-6 w-6"
+                      fill="none"
+                      viewBox="0 0 24 24"
+                      stroke="currentColor"
+                    >
+                      <path
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
+                        stroke-width="2"
+                        d="M3 8l7.89 5.26a2 2 0 002.22 0L21 8M5 19h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v10a2 2 0 002 2z"
+                      />
+                    </svg>
+                  </p>
+                </li>
+                <li>
+                  <ion-icon name="bookmark-outline" class="list"></ion-icon>
+                </li>
+
+                <li>
+                  <ion-icon name="list-outline" class="list"></ion-icon>
+                </li>
+                <li>
+                  <ion-icon name="person-outline" class="list"></ion-icon>
+                </li>
+                <li>
+                  <ion-icon
+                    name="ellipsis-horizontal-circle-outline"
+                    class="list"
+                  ></ion-icon>
+                </li>
+              </ul>
+            </div>
+            <ul class="font-home-list-whats-h list list-clr">
+              <li class="">Home</li>
+              <li>Explore</li>
+              <li>Notifications</li>
+              <li>Messages</li>
+              <li>Bookmarks</li>
+              <li>Lists</li>
+              <li>Profile</li>
+              <li>More</li>
+            </ul>
+          </div>
+
+          <button class="l-menu-twt-btn">Tweet</button>
+        </div>
+
+        <div class="lft-btm--e1-b">
+          <div class="e1-b el-1">
+            <img
+              src="/images/img/burningcars.jpg"
+              alt="picture"
+              class="pic-profile"
+            />
+          </div>
+
+          <div class="e1-b e2-2">
+            <ion-icon name="language-outline"></ion-icon
+            ><ion-icon name="language-outline"></ion-icon
```

Comment:
Be careful about having your closing `>` on a newline. It''s a little hard to read. It still works, but difficult for us to read over :)